google-cloud-bigtable-stats/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-bigtable-parent</artifactId>
+        <version>2.2.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- This module is experimental. It is a workaround to support and publish built-in metrics to cloud monitoring
+    through Stackdriver. Built-in metrics will be implemented with shaded OpenCensus so it won't interfere with
+    customer's application metrics. -->
+    <artifactId>google-cloud-bigtable-stats</artifactId>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.google.cloud</groupId>
+                <artifactId>google-cloud-bigtable-deps-bom</artifactId>
+                <version>2.2.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-bigtable:current} -->
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-impl</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadedArtifactAttached>false</shadedArtifactAttached>
+                            <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+                            <artifactSet>
+                                <includes>
+                                    <include>io.opencensus:*</include>
+                                </includes>
+                            </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>io.opencensus</pattern>
+                                    <shadedPattern>
+                                        com.google.bigtable.repackaged.io.opencensus
```

Comment:
can you make sure this doesnt collide with java-bigtable-hbase?

Reply:
looks like hbase uses the same shadedPattern: https://github.com/googleapis/java-bigtable-hbase/blob/main/bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml#L245 so I updated it to com.google.bigtable.veneer.repackaged.io.opencensus